msal/tests/dev apps/XForms/XForms.Android/XForms.Android.csproj

```diff
@@ -28,7 +28,7 @@
     <DebugType>full</DebugType>
     <Optimize>false</Optimize>
     <OutputPath>bin\Debug\</OutputPath>
-    <DefineConstants>TRACE;DEBUG</DefineConstants>
+    <DefineConstants>TRACE;DEBUG;ANDROID</DefineConstants>
```

Comment:
><DefineConstants>TRACE;DEBUG;ANDROID</DefineConstants> [](start = 4, length = 54)

move the definition of ANDROID the PropertyGroup above as that will ensure all configurations has it set.

Reply:
same for iOS

---
In reply to: [189373909](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1057#discussion_r189373909) [](ancestors = 189373909)

Reply:
I was having issues getting the individual apps to pick up the ifdefs in UIParent, and found this as a solution...but now, i tried without and they work fine, so am going to remove this.

---
In reply to: [189373909](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1057#discussion_r189373909) [](ancestors = 189373909)